packages/sync/src/class-actions.php

```diff
@@ -757,10 +758,14 @@ public static function get_sync_status( $fields = null ) {
 				'queue_size'           => $queue->size(),
 				'queue_lag'            => $queue->lag(),
 				'queue_next_sync'      => ( self::$sender->get_next_sync_time( 'sync' ) - microtime( true ) ),
-				'full_queue_size'      => $full_queue->size(),
-				'full_queue_lag'       => $full_queue->lag(),
 				'full_queue_next_sync' => ( self::$sender->get_next_sync_time( 'full_sync' ) - microtime( true ) ),
 			)
 		);
+
+		if ( ! strpos( get_class( $sync_module ), 'Full_Sync_Immediately' ) ) {
```

Comment:
this Full_Sync_Immeditaley class is super temporary and it will replace the current one once we validate it works, famous last words.

I agree about doing a better usage of `strpos`
```suggestion
		if ( false === strpos( get_class( $sync_module ), 'Full_Sync_Immediately' ) ) {
```